deploy/test/test_cases/TEST_ID_20_helm_multiple_provider_same_serviceaccount.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that two Secrets Provider Jobs can run with same Service Account successfully in the same namespace
```

Comment:
@oburstein-hub @eladkug @orenbm please see the new test to cover the flow where the customer uses the same Service Account for both Secrets Providers